plugins/catalog-react/src/components/EntityRefLink/EntityRefLinks.tsx

```diff
@@ -27,6 +26,7 @@ import { LinkProps } from '@backstage/core-components';
 export type EntityRefLinksProps = {
   entityRefs: (string | Entity | CompoundEntityRef)[];
   defaultKind?: string;
+  titleFn?: (r: string | Entity | CompoundEntityRef) => string;
```

Comment:
It's quite a lot of work for the implementer of this callback to be handling all of these different formats. I'm thinking we'd at least want to normalize it to `CompoundEntityRef`?

Regarding the name, I think MUI generally prefixes this kind of prop with `get*`, so in that case this would be `getTitle`?

Reply:
Sure, we can change it to `get*` format.
Having just `CompoundEntityRef` should be fine too.
Interesting thing, that TypeScript complains to a single implementation only if to do inline function implementation, but if to create a separate function, there is no type errors.

![Screenshot 2022-09-19 at 18 05 25](https://user-images.githubusercontent.com/2265094/191062435-e7da8c9d-67b9-43f3-902e-2612d7d6cfee.png)


Will apply these suggestions shortly.

Reply:
Done

Reply:
Testing, actually React doesn't like such props in components

![Screenshot 2022-09-19 at 18 34 31](https://user-images.githubusercontent.com/2265094/191067819-c6f40218-1301-4f5e-8c5f-b7cb4d9187b3.png)

Never seen it, maybe cause never named it like that before 😃